package/gluon-state/files/lib/gluon/state/check.d/has_default_gw6

```diff
@@ -0,0 +1,2 @@
+#!/bin/ash
```

Comment:
Just use `/bin/sh`, OpenWrt's scripts never specify ash explicitly.

Reply:
Resolved in 270f1bb5e06377eabccc62aa6400dde3d537e9c4